5.5/contrib/common.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Explicitly set the home directory to allow arbitrary uids to use this image
+export HOME=/var/lib/mysql
```

Comment:
hmm...what will this mean for docker exec type operations?  they'll have no homedir, which also means the .bashrc won't be processed, which means they won't have the SCL stuff enabled either.

short of creating the passwd file, i'm not sure there's anything else we can do, but.. something to think about.